app/controllers/judgements_controller.rb

```diff
@@ -7,9 +7,14 @@ def index
     @evaluations = Evaluation.where(date: semester).page(params[:page]).decorate
   end
 
-
-  def edit
+  def show
     @evaluation = Evaluation.find(params[:evaluation_id])
+    @point_details = PointDetail.includes(:point_request)
+      .select { |pd| pd.point_request.evaluation == @evaluation }
+    @evaluation_messages = EvaluationMessage.where(group: @evaluation.group)
```

Comment:
This loads all messages for the given group, and results in 100+ messages for some older groups. I would consider limiting it to the current semester.